web-common/src/features/dashboards/dimension-table/DimensionDisplay.svelte

```diff
@@ -377,28 +376,32 @@
 </script>
 
 {#if topListQuery}
-  <DimensionContainer>
-    <DimensionHeader
-      {metricViewName}
-      {dimensionName}
-      {excludeMode}
-      isFetching={$topListQuery?.isFetching}
-      on:search={(event) => {
-        searchText = event.detail;
-      }}
-    />
-
-    {#if values && columns.length}
-      <DimensionTable
-        on:select-item={(event) => onSelectItem(event)}
-        on:sort={(event) => onSortByColumn(event)}
-        dimensionName={dimension?.name}
-        {columns}
-        {selectedValues}
-        rows={values}
-        {sortByColumn}
+  <div class="h-full flex flex-col" style:min-width="365px">
```

Comment:
All these style directives can be more concise with Tailwind, e.g. `min-w-[365px]`